firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -61,13 +74,52 @@
       nativeComponent = new MissingNativeComponent();
     }
 
+    final Onboarding onboarding = new Onboarding(app, context, idManager, arbiter);
+
+    final BreadcrumbSource breadcrumbSource;
+    final AnalyticsEventLogger analyticsEventLogger;
+
+    if (analyticsConnector != null) {
```

Comment:
This might be a good place for a comment describing the overall approach to the analytics integration, or at least point to one of the class javadocs that describes it.

Reply:
Happy to.